eng/pipelines/templates/steps/sync-directory.yml

```
@@ -25,6 +25,18 @@ steps:
 
       displayName: Copy ${{ parameters.DirectoryToSync }} from azure-sdk-tools to ${{ repo }}
       workingDirectory: $(System.DefaultWorkingDirectory)
+    
+    - ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+      - task: PowerShell@2
+        displayName: 'Get alias of the Tools PR Creator'
+        inputs:
+          targetType: filePath
+          filePath: $(Build.SourcesDirectory)/eng/common/scripts/get-pr-creator.ps1
+          arguments: >
+            -PullRequestNumber "$(System.PullRequest.PullRequestNumber)"
+            -VsoPRCreatorVariable "PullRequestCreator"
```

```suggestion
            -VsoPRCreatorVariable "System.PullRequest.Creator"
```
Lets try to be consistent and use the namespaces something like that.